src/CredentialsManagement.h

```diff
@@ -33,66 +36,129 @@ class CredentialsManagement : public QWidget
 {
     Q_OBJECT
 
-public:
+public:    
+    //-------------------------------------------------------------------------------------------------
+    // Constructor & destructor
+    //-------------------------------------------------------------------------------------------------
```

Comment:
PLease... we all know what a constructor and a destructor is. there is no need to comment those...